zetaclient/bitcoin/btc_signer.go

```diff
@@ -1,4 +1,4 @@
-package zetaclient
+package bitcoin
```

Comment:
We have `bitcoin_client` and `btc_signer`, I think it would be good to make both consistent

Ideally using `bitcoin_signer` I think